packages/ResizeDetector/stories/ResizeDetector.examples.stories.js

```diff
@@ -72,7 +72,7 @@ callbacks.story = {
 export const debounce = () => (
   <ExampleStory storyName="Debounce Examples" component="ResizeDetector" fileName="examples/Debounce.js">
     <p>
-      <code>debounceDelay = 300</code>
+      <code>debounceDelay = 30</code>
```

Comment:
It's deliberately slooooow (300)
https://github.com/acl-services/paprika/blob/master/packages/ResizeDetector/stories/examples/Debounce.js#L9